### PR TITLE
TDC-1373 : Common lib dyn-log dataset to daikon

### PR DESCRIPTION
**Pull request by YoannAlbert**
**What is the problem this Pull Request is trying to solve?**
Add dynamic-log from dataset to daikon, to create a library wich use by other project.

**What is the chosen solution to this problem?**
 
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDC-1373

 
**Please check if the Pull Request fulfills these requirements**
- [X] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/24/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/24/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/25/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/25/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/26/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/26/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/29/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/29/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/31/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/31/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/33/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/33/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/34/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/34/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/35/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/35/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/36/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/36/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/37/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/37/)

**Comment by sgandon** [comment]
@YoannAlbert Could you fill the PR description please ?

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/46/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/46/)

File: daikon-dynamic-log/dynamic-log/src/main/scala/org/talend/daikon/dynamic/config/UiSpecKey.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package org.talend.daikon.dynamic.config
```
Comment (sgandon) [inline daikon-dynamic-log/dynamic-log/src/main/scala/org/talend/daikon/dynamic/config/UiSpecKey.scala]: I don't understand why you would have UI Spec keys inside a module for dynamic logging  ????

File: daikon-dynamic-log/dynamic-log/src/main/scala/org/talend/daikon/dynamic/config/ConfigurationServiceLike.scala [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    *
+    * {appName} being the application identifier, such as dataset or datastreams. Akka persistence configuration
+    * is not the same between those applications since they should use their own database setup so separated
+    * configuration keys are needed.
+    *
+    * @param configuration the [[Configuration]] to transform
+    * @param tenantId      the tenant identifier
+    * @return a tenant aware [[Configuration]]
+    */
+  def toTenantConfiguration(configuration: Configuration, tenantId: String): Configuration = {
```
Comment (sgandon) [inline daikon-dynamic-log/dynamic-log/src/main/scala/org/talend/daikon/dynamic/config/ConfigurationServiceLike.scala]: Is it related to logging ?

**Comment by armeltd** [comment]
replaced by : https://github.com/Talend/daikon/pull/426

**Pull request closed without merging**